src/sidebar/components/FilterStatus.js

```diff
@@ -70,43 +70,46 @@ function FilterStatusPanel({
 }) {
   const store = useStoreProxy();
   return (
-    <div className="FilterStatus">
-      {store.isLoading() ? (
-        <div className="hyp-u-layout-row--center">
+    <Card classes="mb-3 p-3">
+      <div className="flex items-center justify-center space-x-1">
```

Comment:
Indent changed because I moved one containing `div` outside of the conditional; only `classNames` changed in the following diff